src/com/workspace/io/ConsoleWriter.java

```diff
@@ -0,0 +1,7 @@
+package com.workspace.io;
+
+public class ConsoleWriter {
```

Comment:
You can have all these low-level classes extend from Abstract Writer base class. We need need some other types of logging i.e. db logging. At that point of time, we can extend it.

Reply:
right. this will be taken up in next iteration